artifactory/services/utils/aqlquerybuilder.go

```diff
@@ -239,12 +245,16 @@ func buildInnerArchiveQueryPart(triple RepoPathFile, archivePath, archiveName st
 	return fmt.Sprintf(innerQueryPattern, getAqlValue(triple.repo), getAqlValue(triple.path), getAqlValue(triple.file), getAqlValue(archivePath), getAqlValue(archiveName))
 }
 
-func buildExcludeQueryPart(params *CommonParams, useLocalPath, recursive bool) string {
+func buildExcludeQueryPart(params *CommonParams, useLocalPath, recursive bool) (string, error) {
 	excludeQuery := ""
 	var excludeTriples []RepoPathFile
 	if len(params.GetExclusions()) > 0 {
```

Comment:
This if statement seems to redundant, because the below loop will zero interruptions if ```params.GetExclusions()``` is zero.